syntax_and_semantics/literals/char.md

```diff
@@ -13,25 +12,27 @@ It is created by enclosing an UTF-8 character in single quotes.
 'あ'
 ```
 
-You can use a backslash to denote some special characters:
+A backslash denotes a special character, which can either be a named escape sequence or a numerical representation of a unicode codepoint.
 
+Available escape sequences:
 ```crystal
 '\'' # single quote
 '\\' # backslash
+'\b' # backspace
 '\e' # escape
 '\f' # form feed
 '\n' # newline
 '\r' # carriage return
 '\t' # tab
 '\v' # vertical tab
+'\uNNNN' # hexadecimal unicode character
+'\u{NNNN...}' # hexadecimal unicode character
```

Comment:
what is `...` for? I can understand its use for strings, where you can chain unicode chars like `\u{12 13 14}` But I don't think it's useful for the `CHar` doc.

Reply:
True, it should perhaps be described differently. For char literals, it can still be one to six digits. So maybe `\u{N+}` to mimic some sort of regex quantifier?